packages/x-charts-pro/src/internals/material/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { ChartsBaseSlots, ChartsIconSlots } from '@mui/x-charts/models';
+import { defaultSlotsMaterial as communityDefaultSlotsMaterial } from '@mui/x-charts/internals';
+import { ChartsBaseSlotPropsPro } from '../slots/chartsBaseSlots';
+import { ChartsIconSlotPropsPro, ChartsIconSlotsPro } from '../slots/chartsIconSlots';
+import { ChartsExportIcon, ChartsZoomInIcon, ChartsZoomOutIcon } from './icons';
+import { BaseDivider } from './components/BaseDivider';
+import { BaseMenuItem } from './components/BaseMenuItem';
+import { BaseMenuList } from './components/BaseMenuList';
+import { BasePopper } from './components/BasePopper';
+import { BaseTooltip } from './components/BaseTooltip';
+
+// Ensures the module augmentation is applied correctly
+// eslint-disable-next-line no-restricted-imports
+import type {} from '@mui/x-charts';
+
+const baseSlots: Omit<ChartsBaseSlots, keyof ChartsBaseSlots> = {
```

Comment:
This is used in multiple places, but feel wrong `Omit<ChartsBaseSlots, keyof ChartsBaseSlots>` = any object.

https://www.typescriptlang.org/play/?#code/FAFwngDgpgBAKlAziGBeGBvYMcwGYCWATsgFwwB2ArgLYBGURA3NrolAMYD2FAJuciIEKAcxYBfYKEiwEyAExoYAeRoEQAHjkgANDADWUMFzzwkIAHxTuFZDBDlti9Flz5iZGAHIu+rztYcdht+eyIqKGBxIA